routes/mods.py

```diff
@@ -129,9 +130,25 @@ def dict_all(models):
 
         authors.append({"id": user_id, "role": AuthorRole.Owner})
 
+        if is_private_beta is not None:
+            mod.is_private_beta = is_private_beta
+
+        if playtesters is not None:
+            for playtester in playtesters:
```

Comment:
Could probably just ditch the `unique_id` list here and use `seen` in future, as it'd be unique, or ditch the seen set and use `unique_id`. Not really any need for 2 things.